frontend/src/components/CreateDataSourceExample.vue

```diff
@@ -155,7 +155,7 @@ const grantStatement = (
   if (dataSourceType === "ADMIN") {
     switch (engineType) {
       case "MYSQL":
-        return "CREATE USER bytebase@'%' IDENTIFIED BY 'YOUR_DB_PWD';\n\nGRANT ALTER, ALTER ROUTINE, CREATE, CREATE ROUTINE, CREATE VIEW, \nDELETE, DROP, EVENT, EXECUTE, INDEX, INSERT, PROCESS, REFERENCES, \nSELECT, SHOW DATABASES, SHOW VIEW, TRIGGER, UPDATE, USAGE, \nFLUSH_TABLES, LOCK TABLES, REPLICATION CLIENT, REPLICATION SLAVE, \nREPLICATION_APPLIER, SESSION_VARIABLES_ADMIN \nON *.* to bytebase@'%';";
+        return "CREATE USER bytebase@'%' IDENTIFIED BY 'YOUR_DB_PWD';\n\nGRANT ALTER, ALTER ROUTINE, CREATE, CREATE ROUTINE, CREATE VIEW, \nDELETE, DROP, EVENT, EXECUTE, INDEX, INSERT, PROCESS, REFERENCES, \nSELECT, SHOW DATABASES, SHOW VIEW, TRIGGER, UPDATE, USAGE, \nLOCK TABLES, REPLICATION CLIENT, REPLICATION SLAVE \nON *.* to bytebase@'%';";
```

Comment:
I think these are used for PITR and gh-ost at least FLUSH_TABLES.

Reply:
Good catch